www/artists/mbolen/index.html

```diff
@@ -13,6 +13,7 @@
         <link href="css/stylish-portfolio.css" rel="stylesheet"><!-- Custom Fonts -->
         <link href="font-awesome/css/font-awesome.min.css" rel="stylesheet" type="text/css">
         <link href="css/onepage-scroll.css" rel="stylesheet" type="text/css">
+		<link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3mobile.css"><!-- Mobile -->
```

Comment:
This has inconsistent spacing

Reply:
I already have that enabled and clearly, it did not work. Which one will work? (tell me the number)
![2018-08-22_21-06-30](https://user-images.githubusercontent.com/7843719/44502549-45e40a00-a64f-11e8-9d0d-8c0673c66ac7.png)